app/src/hooks/projectshook.ts

```diff
@@ -12,7 +12,7 @@ export const useProjectsHook = () => {
   const getProjectById = async (projectId: string) => {
     try {
       setFetching(true);
-      const response: Project = await AxiosPublic.get(`/project/${projectId}`);
+      const response: Project = await AxiosPublic.get(`/api/v1/project/${projectId}`);
```

Comment:
This should really be parameterised somewhere. Can you add a base URL to AxiosPublic or a constant somewhere?

Reply:
This had been set in the axios config through an env variable previously.  Not sure if that changed.

Reply:
`REACT_APP_BASE_API_URL=http://localhost:4000/api/v1`

Reply:
Future task will be to make these consistent and set in a single place. REACT_APP_BASE_API_URL makes sense as a single place to set this.